src/js/helpers/location.js

```diff
@@ -0,0 +1,13 @@
+export const getEncodedLocation = location => {
```

Comment:
this is purely for sticking address fields together?

Reply:
it's what we were already using for our staticmap links, so I moved it to a helper file